Automattic/wp-calypso
# Hide cookie banner for WP apps

#### Changes proposed in this Pull Request
* Hide the cookie banner on requests coming from the mobile app (we're already in the app, so we don't need to show this banner).

<img width="991" alt="Screen Shot 2019-03-28 at 3 41 31 PM" src="https://user-images.githubusercontent.com/1398304/55166799-114da900-5170-11e9-9236-fa1dd95a69ed.png">

#### Testing instructions

* Open the reader in the calypso.live environment of this PR (you won't have dismissed the cookie banner there)
* Use a fake user agent that contains either `wp-iphone` or `wp-android`:
  * iOS example: `Mozilla/5.0 (iPhone; CPU iPhone OS 12_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/16B91 wp-iphone/12.1`.
  * Android example: `Mozilla/5.0 (Linux; Android 6.0; Android SDK built for x86_64 Build/MASTER; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/44.0.2403.119 Mobile Safari/537.36 wp-android/4.7`.
  * On Chrome you can do this using the Network conditions on the browser devtools:
<img width="404" alt="Screen Shot 2019-03-28 at 13 01 49" src="https://user-images.githubusercontent.com/1233880/55129228-ab562700-5159-11e9-9a75-d05cfe97b65e.png">
<img width="900" alt="Screen Shot 2019-03-28 at 13 02 45" src="https://user-images.githubusercontent.com/1233880/55129288-df314c80-5159-11e9-8278-f14471c9eb2f.png">

* Reload the page.
* Make sure the cookie banner doesn't show up now.

Fixes #31158.
